### ISSUE TITLE
New item - Half-width angled panels

### ISSUE THREAD

**Issue by snorepion** [BEEmod/BEE2-items#1973]
Being able to use a half-width (rather than half-length) angled panel would be immensely helpful for test chambers with parts like this: 
![hwpanel](https://cloud.githubusercontent.com/assets/24462989/26224908/c8d52dc4-3be1-11e7-831a-488ecfcd6145.PNG)
You can't fit regular-sized angled panels on a voxel with even one glass panel, let alone two like this. However, I need to make the player be able to shorten the distance that the laser goes, and since angled panels can't be used here, I can't really do that. 
Animation for half-length static panels had been suggested and one of the devs said they could just copy the animations from full-sized panels. The same thing would apply here.

**Comment by TeamSpen210** [comment]
I actually have already cut up a angled panel in this shape, for another item. This is definitely doable.

**Comment by snorepion** [comment]
Perfect! Can you put it in the next release or in one of the next few releases?

**Comment by LautaroL20** [comment]
The closing commit is pointing to the incorrect issue and eventually will be going to wrongly close this suggestion, can this be corrected?

**Comment by TeamSpen210** [comment]
Yep, I immediately re-committed that. Actually this is the only place you can even see the commit.
